src/tss2-tcti/tcti-py.h

```diff
@@ -0,0 +1,36 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#ifndef TCTI_CMD_H
+#define TCTI_CMD_H
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <limits.h>
+
+#include <Python.h>
+
+#include "tcti-common.h"
+#include "util-io/io.h"
+
+#define TCTI_PY_NAME "tcti-py"
+#define TCTI_PY_DESCRIPTION "TCTI module for writing a TCTI in Python3."
+#define TCTI_PY_HELP "String used as module:arg passed to tcti_init(arg: str) -> TCTI Object."
+
+#define TCTI_PY_MAGIC 0xcf5e2c7f4b24deb0ULL
+
+typedef struct TSS2_TCTI_PY_CONTEXT TSS2_TCTI_PY_CONTEXT;
+struct TSS2_TCTI_PY_CONTEXT {
+    TSS2_TCTI_COMMON_CONTEXT common;
+    PyObject *py_tcti;
+    PyObject *cur_response;
+    struct {
+        /* no finalize as __del__ will be called for the TCTI instance */
+        PyObject *transmit;
+        PyObject *receive;
+        PyObject *make_sticky;
+    } methods;
```

Comment:
Well I dropped these do to a weird bug, but I spent all day figuring it out, It was all from not clearing the error handler when get_attr() in the C code failed. Later I would get errors when calling the next Py_<function> but it was related to something N calls ago :-p. It was wierd that when you called transmit, it would say, no attribute named 'make_sticky'.